extensions/positron-python/src/test/common/configSettings/configSettings.unit.test.ts

```diff
@@ -240,7 +245,51 @@ suite('Python Settings', async () => {
             testLanguageServer(v.ls, v.expected, v.default);
         });
 
-        testLanguageServer('invalid' as LanguageServerType, LanguageServerType.None, true);
+        testLanguageServer('invalid' as LanguageServerType, LanguageServerType.Jedi, true);
+    });
+
+    function testPyreflySettings(pyreflyInstalled: boolean, pyreflyDisabled: boolean, languageServerDisabled: boolean) {
+        test(`pyrefly ${pyreflyInstalled ? 'installed' : 'not installed'} and ${
+            pyreflyDisabled ? 'disabled' : 'enabled'
+        }`, () => {
+            if (pyreflyInstalled) {
+                extensions.extensionIdsToFind = ['meta.pyrefly'];
+            } else {
+                extensions.extensionIdsToFind = [];
+            }
+            config.setup((c) => c.get<boolean>('pyrefly.disableLanguageServices')).returns(() => pyreflyDisabled);
+
+            config
+                .setup((c) => c.get<string>('languageServer'))
+                .returns(() => undefined)
+                .verifiable(TypeMoq.Times.once());
+
+            settings.update(config.object);
+
+            if (languageServerDisabled) {
+                expect(settings.languageServer).to.equal(LanguageServerType.None);
+            } else {
+                expect(settings.languageServer).not.to.equal(LanguageServerType.None);
+            }
+            expect(settings.languageServerIsDefault).to.equal(true);
+            config.verifyAll();
+        });
+    }
+
+    // --- Start Positron ---
+    // We are using Jedi as the default language server, and it is not configurable.
+    suite.skip('pyrefly languageServer settings', async () => {
```

Comment:
we're skipping pyrefly tests for now, since the LSP is not configurable in Positron